x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/cases_table.tsx

```diff
@@ -1,3 +1,4 @@
+cases;
```

Comment:
This seems like a mistake, please revert this change :)

Reply:
yeah. lol linter blowing up on me too :D Thank you